### DISCUSSION TITLE
Small grid in osu!lazer mania editor

### DISCUSSION THREAD

**Discussion by n3vesl1s** [ppy/osu#30104] [Q&A] [answered]
Why is it too small, how I can change this (spaces between notes)
![screenshot](https://github.com/user-attachments/assets/4f749860-1fca-4cf3-a3aa-c98f82212654)

**Comment by n3vesl1s** [comment]
Changing the skin doesn't solve this

**Reply by bdach** [reply, marked as answer]
Zoom in on the timeline.
